src/eval.jl

```diff
@@ -54,7 +66,7 @@ handle("eval") do data
   lock(evallock)
   result = @dynamic let Media.input = Editor()
     withpath(path) do
-      @errs include_string(mod, text, path, line)
+      @errs include_string(mod, text, path, line) ends_with_semicolon(text)
```

Comment:
Yes, @TsurHerman let's apply ends_with_semicolon to the last line only. Note we can use `split(..., keep=false)` as opposed to the filter.